platform/iphone/app_delegate.mm

```diff
@@ -603,111 +125,56 @@ - (void)applicationDidReceiveMemoryWarning:(UIApplication *)application {
 };
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-	CGRect rect = [[UIScreen mainScreen] bounds];
-
-	is_focus_out = false;
-
-	[application setStatusBarHidden:YES withAnimation:UIStatusBarAnimationNone];
-	// disable idle timer
-	// application.idleTimerDisabled = YES;
+	self.focused = NO;
 
 	// Create a full-screen window
-	window = [[UIWindow alloc] initWithFrame:rect];
-	// window.autoresizesSubviews = YES;
-	//[window setAutoresizingMask:UIViewAutoresizingFlexibleWidth |
-	// UIViewAutoresizingFlexibleWidth];
-
-	// Create the OpenGL ES view and add it to the window
-	GLView *glView = [[GLView alloc] initWithFrame:rect];
-	printf("glview is %p\n", glView);
-	//[window addSubview:glView];
-	glView.delegate = self;
-	// glView.autoresizesSubviews = YES;
-	//[glView setAutoresizingMask:UIViewAutoresizingFlexibleWidth |
-	// UIViewAutoresizingFlexibleWidth];
```

Comment:
I tried to leave a lot of the previous trial and error as git history. If any of this is valuable, please let me know.